### PR TITLE
Update lazy_imports_lib.py

### LINKED ISSUE #1696: show_examples fails to import matplotlib.pyplot
**Short description**
`show_examples` leads to an error: "AttributeError: module 'matplotlib' has no attribute 'pyplot'"

**Environment information**
* Operating System: Linux
* Python version: 3.7.4
* `tensorflow-datasets` version: 2.0.0

**Reproduction instructions**

```
import tensorflow_datasets as tfds
datasets, info = tfds.load(name='stanford_dogs',
                               with_info=True, split='train')
tfds.show_examples(
    info, datasets, rows=3, cols=3, plot_scale=3.0, image_key=None
)
```

**Additional context**
https://github.com/tensorflow/datasets/blob/dcaa4c76905bb9eb14b4e0947d1fce15c48aea0f/tensorflow_datasets/core/visualization.py#L66 is not enough, you need to explicitly `import matplotlib.pyplot`, see https://stackoverflow.com/questions/14812342/matplotlib-has-no-attribute-pyplot

### PR DESCRIPTION
**Pull request by Eshan-Agarwal**
Fixes #1696 , when running locally `tfds.show_examples()` shows "AttributeError: module 'matplotlib' has no attribute 'pyplot'"

### CODE REVIEW

File: tensorflow_datasets/core/lazy_imports_lib.py [hunk 1]
```diff
@@ -12,8 +12,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
```
Comment (Conchylicultor) [inline tensorflow_datasets/core/lazy_imports_lib.py]: Why change this ?

File: tensorflow_datasets/core/lazy_imports_lib.py [hunk 2]
```diff
@@ -61,11 +61,6 @@ def crepe(cls):
   def cv2(cls):
     return _try_import("cv2")  # pylint: disable=unreachable
 
-  @utils.classproperty
```
Comment (Conchylicultor) [inline tensorflow_datasets/core/lazy_imports_lib.py]: Why delete this ?

File: tensorflow_datasets/core/lazy_imports_lib.py [hunk 3]
```diff
@@ -12,8 +12,8 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-
 # Lint as: python3
+
```
Comment (Conchylicultor) [inline tensorflow_datasets/core/lazy_imports_lib.py]: Why change this ? This make it inconsistent with other files.

Comment (Eshan-Agarwal) [reply tensorflow_datasets/core/lazy_imports_lib.py]: I am sorry for these changes , I just copy code from my local cloned repo of tfds I can fix them now
Thanks for letting me know

**Comment by Eshan-Agarwal** [comment]
@Conchylicultor  Thanks for your review, please have a look, I correct all things mistakenly done by me

**Pull request merged**